src/Morphir/Visual/ViewList.elm

```diff
@@ -1,25 +1,68 @@
 module Morphir.Visual.ViewList exposing (view)
 
 import Dict
-import Element exposing (Element, centerX, centerY, el, fill, height, indexedTable, none, padding, spacing, table, text, width)
+import Element exposing (Element, centerX, centerY, el, fill, height, indexedTable, none, padding, row, spacing, table, text, width)
+import Element.Background as Background
 import Element.Border as Border
 import Element.Font as Font
 import Morphir.IR as IR
 import Morphir.IR.Name as Name
 import Morphir.IR.Type as Type exposing (Type)
 import Morphir.IR.Value as Value
-import Morphir.Visual.Config exposing (Config)
+import Morphir.Visual.Config exposing (Config, evaluate)
 import Morphir.Visual.EnrichedValue exposing (EnrichedValue)
 import Morphir.Visual.Theme exposing (smallPadding, smallSpacing)
 
 
-view : Config msg -> (EnrichedValue -> Element msg) -> Type () -> List EnrichedValue -> Element msg
-view config viewValue itemType items =
+view : Config msg -> (EnrichedValue -> Element msg) -> Type () -> List EnrichedValue -> Maybe EnrichedValue -> Element msg
+view config viewValue itemType items maybeItemToHighlight =
+    let
+        shouldHighLight : EnrichedValue -> Bool
+        shouldHighLight currentItem =
+            case maybeItemToHighlight of
+                Nothing ->
+                    False
+
+                Just i ->
+                    case
+                        evaluate (Value.toRawValue i) config
+                            |> Result.andThen (\res1 -> evaluate (Value.toRawValue currentItem) config |> Result.andThen (\res2 -> Ok (res1 == res2)))
+                    of
+                        Ok a ->
+                            a
+
+                        Err _ ->
+                            False
+    in
     if List.isEmpty items then
         el [ centerX, centerY ]
             (text " [ ] ")
 
     else
+        let
+            defaultDisplay =
```

Comment:
Can you add the types here? Also, when you define a value like this it will always be evaluated even if it's not shown. You can avoid that by passing in something. For example if this took the `items` as input it would only be evaluated when used.